### DISCUSSION TITLE
TP-Link Tapo C110 Motion detection does not seem to work in Frigate Home Assistant

### DISCUSSION THREAD

**Discussion by koosoli** [blakeblackshear/frigate#5033] [Ask A Question]
Hi, I have 2 IP cameras working perfectly in Home Assistant with the Frigate Addon which runs on mi pi400 without a CoralAI. That means also Human detection works.
However, I did add a TP-Link Tapo C110 to the setup. I copy pasted the configuration from the other working cameras and updated the resolution and path to the new camera (to what I thought was correct without being sure). I can see the new camera in the Frigate UI but it does not seem to detect any motion or humans. What might I have missed? If I did anything wrong with resolution or framerate, would that explain why I could see the camera's image but it would not detect motion or humans?
Thanks

**Comment by koosoli** [comment]
Ok, solved, was able to get the correct FPS and Resolution with ONVIF and I can confirm that if those config values do not match with the real output that Frigate will not detect motion without any errors.

**Reply by NickM-27** [reply]
That is incorrect, you can set the fps to any value and ffmpeg will adjust for that. Same with resolution, there was likely a different issue going on.

**Reply by koosoli** [reply]
Ok, this is good to know! The problem is also not solved. I got 59 % match that a trash can is a person over 70 % that a car is a person and nothing on a real person...

**Reply by NickM-27** [reply]
You should make an actual issue and answer all the prompts, there's no way to help you with such little information

**Comment by danielukm8** [comment]
can you share the configuration with frigate? I'm trying to add a c110 but it seems it keeps falling.
